airflow/www/static/js/dag_code.js

```diff
@@ -17,13 +17,19 @@
  * under the License.
  */
 
-/* global $ */
+/* global window, $ */
 
 import getMetaValue from './meta_value';
 
+function toggleWrap() {
+  $('.code pre').toggleClass('wrap');
+}
+
 const isWrapped = getMetaValue('wrapped');
 
 // pygments generates the HTML so set wrap toggle via js
 if (isWrapped) {
```

Comment:
`isWrapped` is not being translated to a js boolean correctly. Would you mind changing this line?
```suggestion
if (isWrapped === 'True') {
```